en/guide/extending_csd.md

```diff
@@ -1,6 +1,121 @@
 # Extending Camera Streaming Daemon
 
-## Create a Custom Video Stream
+## Create a Custom Camera Device
+
+Camera-Streaming-Daemon has native support for auto detection of video cameras exported as Video4Linux(V4L2) devices. This covers most of regular Linux cameras, but there are some cameras that don't have V4L2 support or cases where user wants to set very specific parameters or do custom post-processing in video before exporting.
+
+For these use cases, it is possible to create a custom CameraDevice class, representing a single custom camera device.
```

Comment:
Just generally, whenever you have a class or other C++ object, mark it up with a single backtick either side. So here CameraDevice should be \`CameraDevice\` (`CameraDevice`).

Where you say "representing a single custom camera device" do you just mean "representing a new type of camera device"?